masquerade-core/src/main/java/com/flipkart/masquerade/util/Helper.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2017 Flipkart Internet, pvt ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.flipkart.masquerade.util;
+
+import com.flipkart.masquerade.Configuration;
+import com.flipkart.masquerade.rule.BasicRule;
+import com.flipkart.masquerade.rule.Rule;
+import com.google.common.reflect.ClassPath;
+import com.squareup.javapoet.ClassName;
+
+import java.io.IOException;
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static com.flipkart.masquerade.util.Strings.*;
+
+/**
+ * Created by shrey.garg on 25/04/17.
+ */
+public class Helper {
+    public static String getSetterName(String name) {
+        String capitalizedName = capitalize(name);
+        String prefix = "set";
+        return prefix + capitalizedName;
+    }
+
+    public static String getGetterName(String name) {
+        String capitalizedName = capitalize(name);
+        String prefix = "get";
+        return prefix + capitalizedName;
```

Comment:
I'll throw an Exception is this case so that the generated code is always compilable